org.eclipse.xtext.tests/testdata/wizard-expectations/full/full.parent/full/build.gradle

```diff
@@ -1,10 +1,10 @@
 dependencies {
-	compile "org.eclipse.xtext:org.eclipse.xtext:${xtextVersion}"
-	compile "org.eclipse.xtext:org.eclipse.xtext.xbase:${xtextVersion}"
+	api "org.eclipse.xtext:org.eclipse.xtext:${xtextVersion}"
+	api "org.eclipse.xtext:org.eclipse.xtext.xbase:${xtextVersion}"
 }
 configurations {
 	mwe2 {
-		extendsFrom compile
+		extendsFrom api
```

Comment:
We should extend from `compileClasspath` here, since that also includes things like `compileOnly`.

Reply:
will give this a try